docs/metrics_v2.md

```diff
@@ -145,3 +145,6 @@ public interface StormMetricsFilter extends MetricFilter {
 }
 ```
 
+V2 metrics can also be reported to the Metrics Consumers registered with topology.metrics.consumer.register by enabling the 
+topology.enable.v2.metrics.tick configuration.
```

Comment:
nit: no newline needed

Reply:
I'm not clear what newline you're referring to.  The one at the end of line 149?  Let me know and I will put up a PR with the other changes.

Reply:
The one at the end of line 148